MAINTAINERS.yml

```diff
@@ -2172,6 +2172,7 @@ Release Notes:
     - rlubos
```

Comment:
No objections to this. Could you change the commit subject to
`MAINTAINERS: Add MaochenWang1 as a collaborator to Wi-Fi`
so that it is easier to see from git log who is being added and to what area.

Reply:
thanks for this suggestion, please review again.